### ISSUE TITLE
Shiny apps not displayed properly in IE

### ISSUE THREAD

**Issue by msenn** [openanalytics/shinyproxy#12]
Hi,

Internet Explorer doesn't display apps properly. The app window doesn't fill the full height of the window but only a narrow bar:

![shinyproxy](https://cloud.githubusercontent.com/assets/15923027/26406772/8eed7f64-4099-11e7-868c-270cc639678a.png)

Speculation: This might be related to the use of `$(window).height()` in the template.
https://stackoverflow.com/questions/5692746/window-height-fails-on-ie9

Thanks for looking into this

**Comment by fmichielssen** [comment]
Hi,

I am currently unable to reproduce this on IE11.
Can you tell me which IE version you are using, and any other settings in IE or ShinyProxy that might be related?

**Comment by ahalasz** [comment]
Hi,

The IE version is 11.0.9600.18697. I am not aware of any particular setting in IE (I disabled all the addins and the issue persisted). Do you have any particular setting in mind? (I honestly don't know what to look for here).

Attached you can find the application.yml we use.
[application.yml.txt](https://github.com/openanalytics/shinyproxy/files/1137965/application.yml.txt)

Thank you again for looking into this.

**Comment by fmichielssen** [comment]
Thanks for the feedback.
I believe this layout issue is fixed in 25f2782e6f8f771b4790cba92108c9ef89b74394
Please let us know if you still see layout quirks!

**Comment by msenn** [comment]
Thanks for looking into this. We'll check it as soon as we my colleague is back from vacation.

**Comment by ahalasz** [comment]
Hi, 
I just installed version 0.9.4 and the issue still persists. 

![untitled](https://user-images.githubusercontent.com/8928800/28882648-20a1dc90-77ac-11e7-8690-bb5c05769078.png)

Thank you for looking into this.

**Comment by fmichielssen** [comment]
Hi @ahalasz ,

On my end, 01_hello looks like this on shinyproxy 0.9.4 and IE 11.483.15063.0:

![helloshinyie](https://user-images.githubusercontent.com/5015362/29021083-a78af924-7b64-11e7-9b6e-abf24f68e138.png)

I.e. I do not see the _Shiny Applications (powered by R)_ text and the app fills the page as expected.
Can you confirm that you have no custom CSS applied, and that no stale CSS is being used (try force refresh using Ctrl+F5)?

Regards,
Frederick

**Comment by ahalasz** [comment]
Hi @fmichielssen,

I confirm that I don't have a custom CSS and I tried force refreshing and did not help. I use the application,yml file that I provided above in one of my posts.

What I noticed is that if I disable the "Display intranet sites in Compatibility View" then the window becomes bigger but not all the elements of the Shiny app are displayed. 

![capture](https://user-images.githubusercontent.com/8928800/29025134-9b6cc486-7b75-11e7-8d44-90c0ac2bb8d2.PNG)

Regards,
Andrei

**Comment by ahalasz** [comment]
Hi @fmichielssen,

I just realised that if I disable the "Display intranet sites in Compatibility View" in IE and I click on the "show below" button in the hello Shiny app, the display is correct. 

![capture](https://user-images.githubusercontent.com/8928800/29026353-abf5a2cc-7b7b-11e7-81a9-9ae493831085.PNG)

I checked some of my other apps and they look also ok. I still don't understand why I need to switch the view in the "hellp" app (I don't need to in Chrome for example) but since my other apps seem to display fine now I suggest we close this issue.

Thank you again for your help,
Andrei
